german/the-black-hack.md

```diff
@@ -46,9 +46,9 @@ Rüstung bietet Schutz, indem sie allen erlittenen Schaden reduziert. Jedes Stü
 
 |     Typ        | Punkte |
 |:--------------:|:------:|
-|  Festes Wams   |   2    |
-|  Lederrüstung  |   4    |
-|   Kettenhemd   |   6    |
+| Gambeson       |   2    |
```

Comment:
Vielleicht habe ich das nicht ganz glücklich ausgedrückt, aber: Nachschlagen kann man wahrscheinlich jedes noch so abstruse Wort im Internet und wahrscheinlich wird es in der Wikipedia dazu irgendeinen Eintrag geben. Das ist aber noch kein Hinweise darauf, dass das Wort allgemein überhaupt bekannt ist. Und ich denke, ein Text, bei dem man andauernd etwas nachschlagen muss, ist nicht sehr attraktiv. Und ich sehe es als Aufgabe eines Übersetzers an, einen Text in die Sprache des Lesers zu übertragen und zwar in die Sprache, die er tatsächlich spricht und die er versteht.

Meiner Meinung nach ist "Gambeson" kein deutsches Wort, ein durchschnittlicher Muttersprachler würde es nicht verstehen. Ich kann es auch in keinem deutschsprachigen Wörterbuch finden. Zu "Wams" hingegen gibt es einen Eintrag im Duden und gibt die Häufigkeit des Worts als unter den Top 100.000 Worten des Dudenkorpus an. Auf dieser Grundlage nehme ich an, dass sich deutlich mehr Leute unter einem "Wams" etwas vorstellen können als unter einem "Gambeson". Wie gesagt, ich lasse mich gern eines Besseren belehren, dass "Gambeson" im normalen Sprachgebrauch geläufig ist oder überhaupt vorkommt. Ich denke auch selbst, dass "Wams" nicht unbedingt ein verbreitetes Wort ist, wir haben ja bereits über bessere Vorschläge nachgedacht.